<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node name="tb_trajectory_writer_node"	pkg="tb_trajectorywriter" type="tb_trajectory_writer_node" respawn="true">
		<param name="translation_euclidian_resolution" type="double" value="1.0" />
		<param name="turnangle_degrees_resolution" 		 type="double" value="1.0" />
</node>
</launch>
